### ISSUE TITLE
[FEATURE] i need printing intermediate output node for debugging

### ISSUE THREAD

**Issue by choigawoon** [FlowiseAI/Flowise#791]
**Describe the feature you'd like**
A clear and concise description of what you would like Flowise to have.

It would be nice if there was a function to output the output of the node.
For example, I want to check if the csv file is being separated well when I put it into the text character splitter.
I get an error when uploading Pinecone, but it's hard to figure out if my document is properly separated.

**Additional context**

**Comment by atishamte** [comment]
In most cases, setting DEBUG=true in the env file gives the performed steps on the terminal.

Did you try this?

**Comment by pancodia** [comment]
How can I see the intermediate output from terminal if I run Flowise using docker compose? 

I thought it is stored in logs. So I accessed the container via interactive shell by using `docker exec -it docker-flowise-1 sh`. Then when I execute the prompt in the UI, I tried to print log in the interactive shell using `tail -f /root/.flowise/logs/server.log `. But I don't see the output mentioned in [the doc](https://docs.flowiseai.com/configuration/environment-variables#debug-and-logs).
